source/glossary.rst

```diff
@@ -520,3 +520,8 @@ Glossary of Terms
      :term:`operating systems <operating system>` without special drivers (on
      Windows, special drivers might still be necessary to use the :term:`ASIO`
      sound :term:`API` though).
+
+   Intro/Outro Markers
```

Comment:
I don't think these need to be mentioned in the glossary.

Reply:
Okay, will remove that.